folly/compression/Compression.cpp

```diff
@@ -69,7 +70,7 @@ namespace io {
 
 Codec::Codec(
     CodecType type,
-    Optional<int> level,
+    const Optional<int>& level,
```

Comment:
This does not seem very helpful.

Reply:
Fair point. In general, it's just checking if a type is trivially copyable. If it is not, it will pass by `const&`. In this case, `Optional` has its own copy constructor. Probably best to just disable that check entirely as it will probably have more false positives than we would like.